batterymonitor@pdcurtis/files/batterymonitor@pdcurtis/po/fr.po

```diff
@@ -0,0 +1,194 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Can you add this PO file to a separate PR instead? Your commits will be squashed and we won't know from the commit history that this happened to a separate xlet. I think its okay if a PR is doing a similar thing to multiple xlets in individual commits and says so, but in this case its not related to adding the other applet.

Reply:
Of course, that's a mistake, sorry. This file will be removed from this branch at the next commit.